# Used by: gate_navigator_node.py

gate_navigator_node:
  ros__parameters:
    # Depth control
    target_depth: -1.5
    depth_correction_gain: 1.0
    depth_tolerance: 0.1
    
    # Speed settings
    search_forward_speed: 0.4
    approach_speed: 0.5
    passing_speed: 0.6
    passing_duration: 5.0
    
    # Alignment
    alignment_threshold: 0.2
    yaw_correction_gain: 0.6
    
    # Distance thresholds
    safe_distance_threshold: 3.0
    passing_distance_threshold: 1.5
    
    # Flare avoidance
    flare_avoidance_gain: 0.8
    flare_avoidance_duration: 3.0
    
    # Timeouts
    gate_lost_timeout: 2.0
    search_timeout: 60.0